dcos/package.py

```diff
@@ -524,9 +524,9 @@ def _extract_default_values(config_schema):
     """
 
     defaults = {}
-    if not config_schema.get('properties'):
+    if 'properties' not in config_schema:
         raise DCOSException("Error with config schema. " +
-                            "Please make sure it's a valid jsonschema.")
+                            "Please contact the package maintainer.")
```

Comment:
Strange.  I thought I already commented on this.

The error message should

a) show the actual error (the malformed config.json)
b) describe what the error is
c) provide the maintainers contact info (it's in the package.json I think)
d) probably say "the package's config.json" rather than the "config schema"